bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/handler/LivisiBridgeHandler.java

```diff
@@ -132,7 +129,17 @@ public LivisiBridgeHandler(final Bridge bridge, final OAuthFactory oAuthFactory,
 
     @Override
     public void handleCommand(final ChannelUID channelUID, final Command command) {
-        // not needed
+        if (CHANNEL_RESTART.equals(channelUID.getId())) {
+            commandRestart(command);
+        } else {
+            logger.debug("UNSUPPORTED channel {} for bridge {}.", channelUID.getId(), bridgeId);
+        }
+    }
+
+    private void commandRestart(Command command) {
+        if (command instanceof OnOffType && OnOffType.ON.equals(command)) {
```

Comment:
Can be simplified to:
```suggestion
        if (OnOffType.ON.equals(command)) {
```

or even:
```suggestion
        if (OnOffType.ON == command) {
```

Reply:
done